csvcubed/csvcubed/cli/entrypoint.py

```diff
@@ -0,0 +1,41 @@
+"""
+CLI
+---
+
+The *Command Line Interface* for `csvcubed inspect`.
+"""
+
+
+import click
+import colorama
+import logging
+
+logging.basicConfig()
+logging.root.setLevel(logging.DEBUG)
+
+from pathlib import Path
+from csvcubed.cli.inspect.inspect import inspect
+
+_logger = logging.getLogger(__name__)
+
+
+@click.group(context_settings=dict(help_option_names=["-h", "--help"]))
+def entry_point():
+    """
+    csvcubed - a tool to generate qb-flavoured CSV-W cubes from COGS-style info.json files.
+    """
+    colorama.init(autoreset=True, wrap=True)
+
+
+# csvcubed build command goes here
+
+
+@entry_point.command("inspect")
```

Comment:
@MuazzamChaud is taking in logging level as a parameter to the command. It might be a good idea to do the same for the `inspect` command. 

Ideally I suppose, it'd be nicer if we could capture & configure logging at the `entry_point` level, but I'm not sure whether that's possible with click or whether we just have to add the same param on to both commands and set logging within it.

Reply:
@robons Please let me know if this comment is still valid. If so, shall I do the update to the entrypoint.py as part of the next PR I am working on currently?

Reply:
So long as your inspect command handles configuring the logging level you don't need to do anything to unify them immedidately. If the inspect command doesn't support logging we need to implement it identically to how it is implemented for the `build` command in `main` now.